express-static/server.js

```diff
@@ -0,0 +1,8 @@
+import express from 'express';
+
+const app = express();
+app.listen(8080, () => {
+  console.log('Express server listening on port 8080');
+});
+
+app.use(express.static('public'));
```

Comment:
While it doesn't make a large difference here, the larger a server gets, the more important it becomes for all the server's middleware to be set up before it starts listening on a network port.